migrations/v6-MIGRATION.md

```diff
@@ -0,0 +1,68 @@
+## v6 API Changes
```

Comment:
Should this be `v5` since in react native this is the migration from v4 to v5?

Reply:
Isn't this a migration from v5 to v6? 🤔 The latest version of RN SDK is `5.13.3`

Reply:
🤦 I believe I got confused here... Not sure what I was thinking. You're right, ignore my comments!